template/web/next.config.js

```diff
@@ -21,6 +21,18 @@ const nextConfig = withPWA({
         protocol: 'https',
         hostname: 'ipfs.io',
       },
+      {
+        protocol: 'https',
+        hostname: 'cloudflare-ipfs.com',
+      },
+      {
+        protocol: 'https',
+        hostname: 'gateway.pinata.cloud',
+      },
+      {
+        protocol: 'https',
+        hostname: 'nftstorage.link',
```

Comment:
Oh that's dope. Teach me a bit, how do they work together?

Reply:
This is mostly telling nextjs to allow to load images from these domains in nextjs NextImage component